test/integration/targets/var_templating/runme.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# this should succeed since we override the undefined variable
+ansible-playbook undefined.yml -i inventory -v "$@" -e '{"mytest": False}'
```

Comment:
It should succeed even if we don't override the undefined variable.

Reply:
yes, that test is below